src/test/java/uk/gov/justice/hmpps/prison/api/resource/impl/OffenderResourceNewBookingTest.kt

```diff
@@ -119,7 +114,7 @@ class OffenderResourceNewBookingTest : ResourceTest() {
 
       @Test
       internal fun `400 when offender is inactive but not OUT, for instance currently being transferred`() {
-        val offenderNo = createActiveBooking(prisonId = "MDI").also { transferOut(it) }
+        val offenderNo = createActiveBooking(prisonId = "MDI").also { builderContext.transferOut(it) }
 
```

Comment:
minor issue; I called the object `builderContext`   - I think I will rename to `testDataContext`  or something since we are not building anything in this scenario just reading data